cluster-autoscaler/core/scale_up.go

```diff
@@ -41,12 +42,12 @@ import (
 // false if it didn't and error if an error occurred. Assumes that all nodes in the cluster are
 // ready and in sync with instance groups.
 func ScaleUp(context *context.AutoscalingContext, processors *ca_processors.AutoscalingProcessors, clusterStateRegistry *clusterstate.ClusterStateRegistry, unschedulablePods []*apiv1.Pod,
-	nodes []*apiv1.Node, daemonSets []*extensionsv1.DaemonSet) (bool, errors.AutoscalerError) {
+	nodes []*apiv1.Node, daemonSets []*extensionsv1.DaemonSet) (bool, *status.ScaleUpStatus, errors.AutoscalerError) {
```

Comment:
We probably don't need the bool return value anymore now that we have ScaleUpStatus

Reply:
Done